.github/workflows/build.main.kts

```diff
@@ -19,6 +20,10 @@ workflow(
         PullRequest(),
     ),
     sourceFile = __FILE__.toPath(),
+    concurrency = Concurrency(
+        group = "\${{ github.workflow }} @ \${{ github.event.pull_request.head.label || github.head_ref || github.ref }}",
```

Comment:
You could make it more type-safe with `expr {}`, but it's a nitpick :)

Reply:
This was a quick copy & paste, so feel free to update it with `expr {}` 👍